integreat_cms/core/settings.py

```diff
@@ -888,6 +883,21 @@
 )
 
 
+#################
+# INTERNAL URLS #
+#################
+
+#: The URLs which are treated as internal in TinyMCE custom link plugin
+INTERNAL_URLS = (
+    ALLOWED_HOSTS
+    + [WEBAPP_URL, WEBSITE_URL]
+    + [
+        x.strip(x)
+        for x in os.environ.get("INTEGREAT_CMS_INTERNAL_URLS", "").splitlines()
+    ]
```

Comment:
```suggestion
        if x
    ]
```
Ah, and sorry for not noticing this earlier: I think the `if x` is required because configparser returns a trailing new line for multi-line config values.

Reply:
Yup! Also noticed that. Sorry for not noticing it in the last suggestion.

I'll have another thorough look and then squash the history.